src/elements/kw-share-feed/kw-share-feed.html

```diff
@@ -58,14 +65,29 @@
             }
         </style>
         <div class="shell" id="shell">
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
+            <!--<kw-share-card share="[[shellShare]]"></kw-share-card>
+            <kw-share-card share="[[shellShare]]"></kw-share-card>-->
+            <kw-share-shell share="[[shellShare]]" slotName="share-card">
```

Comment:
`slotName` is not a valid attribute format see https://www.w3.org/TR/html-markup/syntax.html#syntax-attributes